src/explore-education-statistics-admin/src/pages/release/data/ReleaseApiDataSetDetailsPage.tsx

```diff
@@ -157,16 +183,96 @@ export default function ReleaseApiDataSetDetailsPage() {
+
+  const showDraftVersionTasks =
+    importingStatus !== 'importing' &&
+    (dataSet?.draftVersion?.status === 'Draft' ||
+      dataSet?.draftVersion?.status === 'Mapping');
+
+  const getImportBanner = () => {
+    if (importingStatus === 'importing') {
+      return (
+        <NotificationBanner
+          fullWidthContent
+          heading="Import in progress..."
+          title="Importing"
+        >
+          <p>This process can take a few minutes</p>
+        </NotificationBanner>
+      );
+    }
+
+    if (dataSet?.draftVersion?.status === 'Mapping') {
+      return (
+        <NotificationBanner
+          fullWidthContent
+          heading="This new API data set version has been mapped and is ready to be imported"
+          title="Action required"
+        >
+          <p>
+            If you have completed and checked the new mappings, the next step is
+            to import the changes to enable this version to be published. After
+            importing you will also be able to view the changelog, add guidance
+            notes and also preview this API data set version.
+          </p>
+          <p className="govuk-!-margin-bottom-6">
+            <strong>Please note</strong> you will not be able to make further
+            changes after importing. If you need to make any changes to the
+            mappings you will have to remove this API data set version and
+            create a new version.
+          </p>
+          <Button onClick={handleImport}>Import this data set version</Button>
+        </NotificationBanner>
+      );
     }
-  }
+
+    if (dataSet?.draftVersion?.status === 'Draft') {
+      return (
+        <NotificationBanner
+          fullWidthContent
+          heading="This new API data set version has now been mapped and imported ready for final publication"
+          role={importingStatus === 'imported' ? 'alert' : undefined}
+          title="Mapping import complete"
+          variant={importingStatus === 'imported' ? 'success' : undefined}
+        >
+          <p>
+            If you need to make any changes to the mappings you will have to
+            remove this API data set version and create a new version. You can
+            remove this API data set version up until the release publication.
+          </p>
+          <p>
+            <strong>Please note</strong> these changes will not be made in the
+            public API until the next release has been published.
+          </p>
+
+          <p>
+            <Link to="/todo">View changelog and public guidance notes</Link>
+          </p>
+        </NotificationBanner>
+      );
+    }
+
+    if (
+      importingStatus === 'imported' &&
+      dataSet?.draftVersion?.status === 'Failed'
+    ) {
+      return (
+        <ErrorSummary
+          errors={[
+            {
+              id: 'draft-version-summary',
+              message: 'Data set version import failed',
+            },
+          ]}
+        />
```

Comment:
Do we need to use error summary for this? We're not on a form, so it's a bit weird and inconsistent given we've already been using the notification banner for the other states.

Could just add an error variant of `NotificationBanner`?

Reply:
Changed to use `NotificationBanner` now.